lib/AppInfo/Application.php

```diff
@@ -98,6 +109,40 @@ public function register(IRegistrationContext $context): void
             );
         });
 
+        // Register ObjectEntityMapper with IGroupManager and IUserManager dependencies
+        $context->registerService(ObjectEntityMapper::class, function ($container) {
+            return new ObjectEntityMapper(
+                $container->get('OCP\IDBConnection'),
+                $container->get(MySQLJsonService::class),
+                $container->get('OCP\EventDispatcher\IEventDispatcher'),
+                $container->get('OCP\IUserSession'),
+                $container->get(SchemaMapper::class),
+                $container->get('OCP\IGroupManager'),
+                $container->get('OCP\IUserManager')
+            );
+        });
+
+        // Register ObjectService with IGroupManager and IUserManager dependencies
+        $context->registerService(ObjectService::class, function ($container) {
+            return new ObjectService(
+                $container->get(DeleteObject::class),
+                $container->get(GetObject::class),
+                $container->get(RenderObject::class),
+                $container->get(SaveObject::class),
+                $container->get(ValidateObject::class),
+                $container->get(PublishObject::class),
+                $container->get(DepublishObject::class),
+                $container->get(RegisterMapper::class),
+                $container->get(SchemaMapper::class),
+                $container->get(ObjectEntityMapper::class),
+                $container->get(FileService::class),
+                $container->get('OCP\IUserSession'),
+                $container->get(SearchTrailService::class),
+                $container->get('OCP\IGroupManager'),
+                $container->get('OCP\IUserManager')
+            );
+        });
+
```

Comment:
Why are we registering these services that have always worked without registering them in application.php?

Reply:
Response: somehow in these services something broke, probably something to do with the usermanager and groupmanager